vue-components/src/components/Checkbox.vue

```diff
@@ -9,10 +9,13 @@
 			:checked="checked"
 			:disabled="disabled"
 		>
+		<slot v-if="this.textDirection == 'rtl'" name="suffix" />
```

Comment:
No. We don't want to explicitly specify with props whether this is `ltr` or `rtl`. There are HTML5 technologies that do this for us. The easiest might be flexbox.

Have a look at this commit: https://github.com/wmde/wikit/commit/667cb5b30306ddf5cf2bf23c3ceb64ed4f461c4d

Reply:
looks much cleaner